### ISSUE TITLE
Simple Payments: Fix accessibility by adding appropriate WAI-ARIA tags

### ISSUE THREAD

**Issue by aldavigdis** [Automattic/jetpack#10709]
Simple Payments has a rather nonsemantic markup. This can be mitigated accessibility-wise by using the appropriate WAI-ARIA tags. This should be an easy fix.

See more at https://www.w3.org/WAI/PF/aria/states_and_properties#aria-labelledby

This issue is hereby split from #28781, as that ticket is already out of scope of this one and other things may depend on the current markup style.

**Comment by simison** [comment]
Transferred 👍 

Output HTML is somewhere here:

- https://github.com/Automattic/jetpack/blob/master/modules/widgets/simple-payments.php
- https://github.com/Automattic/jetpack/blob/master/modules/simple-payments/simple-payments.php

**Comment by emilyatmobtown** [comment]
Hi @simison - Is it possible to simulate a paid plan in a Jetpack dev environment? It appears that that's the only way to patch and test this.
